ft/ft/internal.py

```diff
@@ -30,7 +30,7 @@ def ftformat(val):
 
 def try_convert_int(inp):
     # Make sure that the input *only* consists of a number (no added spaces..)
-    if not re.match("^[+-]?\d+$", inp):
+    if not re.match(r"^[+-]?\d+$", inp):
```

Comment:
`\d` was causing a deprecation warning here (it should be either `\\d` or within a raw literal (`r"…"`)